README.md

```diff
@@ -38,6 +38,33 @@ To run the program, simply run:
 python lecturecut.py -h
 ```
 
+### 🐳 Docker
+Docker is a convenient way to build and run LectureCut. Instead of manually installing and maintaining versions of ffmpeg
+and various python libraries on your machine, you can utilize Docker to run LectureCut in as a container.
+Moreover, this repo is expected to change at a fast pace, and so Docker is the easiest way to ensure that you're running
+the most up-to-date version of LectureCut and its dependencies.
+
+#### How it works:
+
+After cloning this repo, you can build the image with this command:
+```bash
+docker build . -t lecturecut
+```
```

Comment:
I think for normal users it would be more convenient to just use the "precompiled" image pushed to ghcr and thus not needing to clone the entire repo. I doubt people want to use a container for dev purposes.

Reply:
That makes sense. I updated this piece with information about pulling from the container registry.